### DISCUSSION TITLE
/sync/watched/shows incorrect last_updated_at

### DISCUSSION THREAD

**Discussion by Tabonx** [trakt/trakt-api#473] [Q&A] [answered]
I've noticed that when calling `/sync/watched/shows` after removing an entry from my watch history, the `last_updated_at` field does not reflect the time of removal. Instead, it shows the date of the last added play. Is this intended behavior, or could it be a bug?

The `/sync/last_activities` endpoint correctly shows the date of the removed play under `episodes/watched_at`. Issue #411 briefly mentions this, but it's unclear if this is the expected behavior. The name `last_updated_at` suggests it should change whenever an update occurs, not just when an entry is added and then rolled back after removal. It would also be more convenient, as I wouldn't need to cache this date for each show and compare them.

@rectifyer

**Comment by rectifyer** [comment, marked as answer]
This is intended behavior. The date is really the last activity date (i.e. watched or collected). We don't track the date on a per show level, just a global date that is used in `last_activities`.
